activity-monitor/activity-monitor.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2014 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
```

Comment:
Please add a file-level comment explaining what this file does.

Also unittests for this and json-logger.go.